MAINTAINERS.md

```diff
@@ -0,0 +1,45 @@
+# What do maintainers do?
+
+Maintainers are people who take an active role in advancing the Baseline Protocol and/or related projects (e.g. the Radish34 Demo). They are primarily responsible for: 
+ - Reviewing and merging PRs into the master branch
+ - Cutting, testing, and releasing new versions of the related Baseline projects
+ - Working with the TSC and SSC to advance the Baseline Protocol
+
+ They can/should also contribute in the following ways:
+ - Writing epics and issues to guide development
+ - Setting up and supporting infrastructure (running demos, CI systems, community projects, etc...) that further Baseline
+ - Working with the community to help with adoption
+ - Presenting the project and key technologies to the public (in-person, webinar, videos, articles, etc...)
+ 
+
+# Who are the current maintainers?
+
+  - [Sam](https://github.com/bitwiseguy) 
+  - [Patrick](https://github.com/pmacom)
+  - [Kartheek](https://github.com/skarred14)
+  - [Brian](https://github.com/breakpointer)
+  - [John](https://github.com/humbitious)
+
+# How to become a maintainer?
+
+To become a maintainer you need to show consistent contributions of code or ideas to the project. This can be in the form of pull requests that get merged into master. Or it can be in the form of issues/planning artifacts that guide the development activities of others. A maintainer needs to be an expert in one or more fields related to the project, show commitment, be reliable, and demonstrate competency.
+
+At the weekly Maintainer Meetings the current members can propose who should be added as a new maintainer and why. The current maintainers must vote with over 2/3rds majority in favor.
+
+# How maintainers organize?
+
+## Slack 
+Maintainers meet and discuss issues virtually via the private #maintainers slack room in the [baseline slack](ethereum-baseline.slack.com). 
+
+## Weekly Meetings (online)
+There are weekly *Maintainers meetings* where members can discuss plans and issues related to the project, updates, release planning, etc... 
+
+# How to stop being a maintainer?
+
+If you are a maintainer you can be voted out if any of the following occur: 
+
+ - You stop reviewing PR's, responding to messages, answering emails, and/or generally ghost the project.
+ - You are disrespectful towards anyone in the community and/or involved in the project in some way.
+ - You violate [the code of conduct](./CODE_OF_CONDUCT.md)
+ 
+A simple majority of maintainers in attendance is required to remove a maintainer, but the TSC may be brought in to arbitrate, if the maintainer to be removed or any other party wishes to dispute the action. Two-thirds of all current maintainers constitute a quorum for a meeting involving a question of removal. 
```

Comment:
I would re-word this. 

Two-thirds of all current maintainers constitute a quorum for a meeting involving a question of removal. A simple majority vote from maintainers attending the meeting is required to remove a maintainer, but the TSC may be brought in to arbitrate if the maintainer to be removed or any other maintainer wishes to dispute the action.

Reply:
I also think that we should mention that if you want to stop being a maintainer you can leave the role at anytime by notifying the other maintainers.